packages/orchestrator-ui-components/src/pages/processes/WfoStartProcessPage.tsx

```diff
@@ -110,17 +115,15 @@ export const WfoStartProcessPage = ({
 
     const submit = useCallback(
         (processInput: object[]) => {
-            const startProcessPromise = apiClient
-                .startProcess(
-                    processName,
-                    startProcessPayload
-                        ? [startProcessPayload, ...processInput]
-                        : [...processInput],
-                )
+            const startProcessPromise = startProcess({
+                workflowName: processName,
+                userInputs: startProcessPayload
```

Comment:
I don't like multiline conditional operator, move this to a const above the `const startProcessPromise`, might fix the multiline.

Reply:
nvm, forgetting its because our small 80 line length...